src/Mod/PartDesign/Gui/TaskChamferParameters.cpp

```diff
@@ -389,10 +417,11 @@ TaskDlgChamferParameters::~TaskDlgChamferParameters()
 //}
 bool TaskDlgChamferParameters::accept()
 {
-    parameter->showObject();
     parameter->apply();
 
     return TaskDlgDressUpParameters::accept();
+    // show only if there was no computation error
+    parameter->showObject();

```

Comment:
This code follows an unconditional return - it won't get called.

Reply:
It will get called, but only when there was no computation error and that's the idea.

The use case is the following:
![pMx0NcIz63](https://user-images.githubusercontent.com/1828501/108456848-94e77600-7271-11eb-839d-54fe1dd1e121.gif)

So when you try to acept an invalid dressup and get an error as return, the broken (actually inexisting dressup) must not be shown.
As you can see this is now the case. And when there is no computation error, the dressup is shown.

Reply:
I am only looking at the code: line 422 is:
```
return TaskDlgDressUpParameters::accept();
```
How can any code on following lines possibly execute?

Reply:
> How can any code on following lines possibly execute?

not at all ;-) Stupid mistake by me. Since the feature is already shown when executing the dialog and later also when  a recompute led to no error, showing it is superfluous and in case there is an error we must not show.

I fixed this now.